pkg/option/config.go

```diff
@@ -433,6 +433,12 @@ const (
 	// ToFQDNsProxyPort is the global port on which the in-agent DNS proxy should listen. Default 0 is a OS-assigned port.
 	ToFQDNsProxyPort = "tofqdns-proxy-port"
 
+	// EnableStandaloneDNSProxy is the name of the option to enable standalone DNS proxy
```

Comment:
I would move these definitions to `service/cell.go` as they don't seem to be used elsewhere.